website/js/index.js

```diff
@@ -466,6 +575,26 @@ var VariantDetail = React.createClass({
                 </Row>
                 <Row>
                     <Col md={8} mdOffset={2}>
+                        { /* eslint-disable dot-notation */ }
+                        <h3>{variant["HGVS_cDNA"]}</h3>
```

Comment:
Please make this change and remove all references to disabling the dot-notation in eslint.